providers/ns1/ns1Provider.go

```diff
@@ -199,6 +204,13 @@ func convert(zr *dns.ZoneRecord, domain string) ([]*models.RecordConfig, error)
 			if err := rec.SetTarget(ans); err != nil {
 				panic(fmt.Errorf("unparsable %s record received from ns1: %w", rtype, err))
 			}
+		case "CAA":
+			//dnscontrol expects quotes around multivalue CAA entries, API doesn't add them, so let's force them
```

Comment:
Please add an integration test that demonstrates this bug.  `integrationTest/integration_test.go` around line 2082/

Reply:
This was actually uncovered with the existing test, `Issue_1374:CAA_spaces` (the testsuite was failing), so I think it's covered.